System/Posix/Files.hsc

```diff
@@ -59,6 +59,11 @@ module System.Posix.Files (
     isBlockDevice, isCharacterDevice, isNamedPipe, isRegularFile,
     isDirectory, isSymbolicLink, isSocket,
 
+#if MIN_VERSION_base(4,10,0)
```

Comment:
You do not need to check this condition any longer, because since #175 `unix.cabal` requires `base-4.10` or newer.

Reply:
Ah! super, thanks :)